packages/redux/src/sharedWishlists/middlewares/index.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Shared wishlists middlewares.
+ */
+export { default as updateSharedWishlistUponItemsChanges } from './updateSharedWishlistUponItemsChanges.js';
```

Comment:
You will need to re-export this in the `packages/redux/src/sharedWishlists/index.ts` file so that it can be available for the consumers to import from the root of the package.

Reply:
Right! Done.